### PR TITLE
Task/xss l n

### PR DESCRIPTION
**Pull request by orthagh**
## Checklist before requesting a review

*Please delete options that are not relevant.*

- [X] I have read the CONTRIBUTING document.
- [X] I have performed a self-review of my code.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] This change requires a documentation update.

## Description

XSS escaping for `src/[L-N]*.php` classes 

+ a few clean (`\n` and comments alignements)

### CODE REVIEW

File: src/Migration.php [hunk 1]
```diff
@@ -129,7 +129,7 @@ public function addNewMessageArea($id)
             echo "<div id='" . htmlspecialchars($this->current_message_area_id) . "'></div>";
         }
 
-        $this->displayMessage(isCommandLine() ? __('Work in progress...') : __s('Work in progress...'));
+        $this->displayMessage(__('Work in progress...'));
```
Comment (trasher) [inline src/Migration.php:132]: I'm not sure this is OK

Comment (orthagh) [reply src/Migration.php:132]: I triple checked this one, the chain goes to displayMessage -> outputMessage -> switch outputMessageToCli or outputMessageToHtml and in the latter, htmlspecialchar is present

File: src/NetworkPortInstantiation.php [hunk 2]
```diff
@@ -716,7 +716,7 @@ public static function dropdownConnect($ID, $options = [])
             $params
         );
 
-        echo "<span id='show_" . $p['name'] . "$rand'>&nbsp;</span>\n";
+        echo "<span id='show_" . $p['name'] . "$rand'>&nbsp;</span>";
```
Comment (trasher) [inline src/NetworkPortInstantiation.php:719]:
```suggestion
        echo "<span id='show_" . htmlspecialchars($p['name']) . "$rand'>&nbsp;</span>";
```

File: src/Migration.php (hunk 1 above)
Comment (trasher) [reply src/Migration.php:132]: :+1:

**Pull request merged**